arrow-site/docs/data/doc-versions.yml

```diff
@@ -0,0 +1,26 @@
+docVersions:
```

Comment:
@israelperezglez , this file is located into `data` instead of `_data`, is it right?

Reply:
The correct site should be _data ;)